CONTRIBUTING.md

```diff
@@ -1,17 +1,195 @@
-# Contributing to Forgerock's Javascript SDK
+# Contribution Guidelines for Ping SDK for Javascript
 
-:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+Welcome! We are excited that you are interested in contributing to the Ping SDK for JavaScript. This document will guide you through the steps required to contribute to the project.
 
-The following is a set of guidelines for contributing to the Forgerock SDK and its packages, which are hosted in the [ForgeRock organization](https://github.com/forgerock) on GitHub.
+The following is a set of guidelines for contributing to the Ping SDK and its packages, which are hosted in the [ForgeRock organization](https://github.com/forgerock) on GitHub.
 
-### Git Flow
+# Setting Up Your Development Environment
 
-Fork the repo to your own github account. From there, you can add the ForgeRock repository as an upstream.
+Before you can contribute to the Ping SDK for Javascript, you'll need to set up your development environment. This section describes the prerequisites and steps needed to start using the project in your local machine.
 
-`git remote add upstream git@github.com:ForgeRock/forgerock-javascript-sdk.git`
+## Prerequisites
+
+- GitHub account.
+- Git installed.
+- (pnpm)[https://pnpm.io/installation]
+- Node 20
+- Setup an Advanced Identity Cloud tenant or PingAM instance, as described in the Documentation.
+
+## Fork and Clone the Repository
+
+Fork the repository to your own GitHub account by clicking the "Fork" button at the top-right of the repository page.
+
+Clone your forked repository to your local machine:
+
+git clone https://github.com/your-username/forgerock-javascript-sdk.git
```

Comment:
Can we mark this line as code? Might make it easier to read, or obvious to copy.